src/types.ts

```diff
@@ -227,6 +227,18 @@ export interface KaboomCtx {
 	scale(): ScaleComp,
 	/**
 	 * Rotation (in degrees).
+	 * 
+	 * @example
+	 * ```js
+	 * // bean WITH anchor: rotates bean based on sprite center 
+	 * // bean WITHOUT anchor: rotates bean relative to local space 
```

Comment:
So I noticed that without anchor("center"), it will rotate the sprite on a different origin, or in this case the default anchor top left, but if I wanted to rotate the sprite without changing it's position at all, I'd use anchor("center"). I will rewrite the explanation to be more clear about that.